lib/Sabberworm/CSS/Parsing/OutputException.php

```diff
@@ -7,12 +7,15 @@
 */
 class OutputException extends \Exception {
     private $iLineNo;
-    public function __construct($sMessage, $iLineNo = 0)
-    {
+    public function __construct($sMessage, $iLineNo = 0) {
         $this->$iLineNo = $iLineNo;
         if (!empty($iLineNo)) {
             $sMessage .= " [line no: $iLineNo]";
         }
         parent::__construct($sMessage);
     }
+
+    public function getLineNo() {
+        return $this->getLine();
```

Comment:
> On 30.06.2016, at 11:37, Sidharth Kshatriya <notifications@github.com> wrote:
> 
> In lib/Sabberworm/CSS/Parsing/OutputException.php:
> 
> >          $this->$iLineNo = $iLineNo;
> >          if (!empty($iLineNo)) {
> >              $sMessage .= " [line no: $iLineNo]";
> >          }
> >          parent::__construct($sMessage);
> >      }
> > +
> > +    public function getLineNo() {
> > +        return $this->getLine();
> 
> @sabberworm Good catch! I've fixed that.
> 
> Anyways, I renamed $iLineNum to $iLineNo everywhere -- is that OK? Or should I revert to the older $iLineNum ?

No, it’s perfect.

> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>